src/gc/objecthandle.cpp

```diff
@@ -776,8 +776,7 @@ bool Ref_InitializeHandleTableBucket(HandleTableBucket* bucket, void* context)
 
         // No free slot.
         // Let's create a new node
-        NewHolder<HandleTableMap> newMap;
-        newMap = new (nothrow) HandleTableMap;
+        HandleTableMap *newMap = new (nothrow) HandleTableMap;
```

Comment:
Why have you removed the holder? It is a pattern that we prefer to use wherever we can.

Reply:
GC is not using holders anywhere else. I do not think it is worth pulling in the whole holder machinery just for the one use.

Reply:
Ah, ok. I have thought we already had it in.